### DISCUSSION TITLE
Error with migrations

### DISCUSSION THREAD

**Discussion by skjorrface** [prisma/prisma#4853] [Q&A] [answered]
Hello, I installed @prisma/cli and @prisma/client ^2.13.1, the command `npx prisma migrate dev --preview-feature` with a simple tutorial schema gives the following error:

`Oops, an unexpected error occured!
Error in migration engine.
Reason: [/root/.cargo/git/checkouts/quaint-9f01e008b9a89c14/bc3aa8a/src/connector/postgres/error.rs:138:29] index out of bounds: the len is 1 but the index is 1`

I'm using Postgresql and debian. What am I doing wrong? Thanks.

**Comment by skjorrface** [comment, marked as answer]
Ok after some investigation I understood it's a database problem; dockerizing a postgresql version 10 instance solved the problem. I don't know what I might be messing up with the instance of postgresql I installed in my machine.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
